diff --git a/config.def.h b/config.def.h
index 877afab..4b29586 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+char font[] = "Sauce Code Powerline:pixelsize=16:antialias=true:autohint=true";
 int borderpx = 2;
 
 /*
@@ -134,7 +134,7 @@ unsigned int cursorshape = 2;
  * Default columns and rows numbers
  */
 
-unsigned int cols = 80;
+unsigned int cols = 150;
 unsigned int rows = 24;
 
 /*
